GetFed/GetFed/Helpers/Identity.swift

```diff
@@ -12,6 +12,10 @@ enum Identity: String {
     case homeVC
     
     case homeToFoodSearchSegue
+    
+    case foodSearchResultCell
+    
+    case FoodResultTableViewCell
```

Comment:
Also, you don't need all of this extra spacing, unless you like that style.

Reply:
I did that to organize the different identifier types listed in the enum. That probably looks strange without context, so I added comments for each identifier category. I hope that's clearer.